README.md

```diff
@@ -1,11 +1,103 @@
+                                 ...
+                                 # Provide an existing test dataset
+                                 test_data=test_dataset,
+                                 ...
+                                 forecasting_parameters=forecasting_parameters)
+    ```
+    Supported input test dataset types: [Dataset](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.dataset.dataset?view=azure-ml-py).
+2. Specifying a train/test percentage split when creating the `AutoMLConfig`.
+   Note, this scenario is not supported for forecasting tasks.
+
+   ```python
+   automl_config = AutoMLConfig(task = 'regression',
+                                ...
+                                # Specify train/test split
+                                training_data=training_data,
+                                test_size=0.2)
+   ```
+   To use a train/test split instead of providing test data directly, use the
+   `test_size` parameter when creating the `AutoMLConfig`. This parameter
+   must be a floating point value between 0.0 and 1.0 and specifies the
+   percentage of the training dataset that should be used for the test
+   dataset. For regression based tasks, random sampling is used. For
+   classification tasks, stratified sampling is used. Forecasting does not
+   currently support specifying a test dataset using a train/test split.
+
+   The `test_data` and `test_size` `AutoMLConfig` parameters are mutually
+   exclusive and can not be specified at the same time.
+
+3. Use `ModelProxy` to test a model after the main AutoML run has completed
+   (aka. "on-demand" model testing).
+
+    ```python
+    from azureml.train.automl.model_proxy import ModelProxy
+
+    model_proxy = ModelProxy(best_run)
+    predictions, metrics = model_proxy.test(test_data)
+    ```
+    Supported input test dataset types: [Dataset](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.dataset.dataset?view=azure-ml-py).
+
+### Get the Predictions and Metrics Generated by the Test Run
+
+When a remote test run is requested by passing in a value for `test_data` or
+`test_size` to `AutoMLConfig`, only the best training run will have a test run
+associated with it.
+
+```python
+best_run, fitted_model = remote_run.get_output()
+test_run = next(best_run.get_children(type='automl.model_test'))
+test_run.wait_for_completion(show_output=False, wait_post_processing=True)
+
+# Get test metrics
+test_run_metrics = test_run.get_metrics()
+for name, value in test_run_metrics.items():
+    print(f"{name}: {value}")
+
+# Get test predictions
+test_run_details = test_run.get_details()
+test_run_predictions = Dataset.get_by_id(workspace, test_run_details['outputDatasets'][0]['identifier']['savedId'])
+test_run_predictions.to_pandas_dataframe().head()
+```
+
+`ModelProxy` already returns the predictions and metrics and
+does not require further processing to retrieve the outputs.
 
 ## Requirements
 
```

Comment:
Is this true? "This datastore is visible to all users with the same subscription. Do not use this feature if any of the information used for or created by the test run needs to remain private."

The datastore and specifically Blog Storage under the covers is secured so ONLY the people with access to the WORKSPACE can have access to it, but not *ANY* user with the same Azure subscription, right?

Reply:
Updated the docs to mention how the workspaces and resource groups can be secured using IAM.